### ISSUE TITLE
[5] X-ViT: High Performance Linear Vision Transformer without Softmax

### ISSUE THREAD

**Issue by techzzt** [techzzt/pytorch-paper-implement#5]
![image](https://user-images.githubusercontent.com/49440852/177768210-3539a0d5-f653-4a3d-b76b-2164f9d7d323.png)

- Computer vision task에서 기존의 self-attention (SA) algorithm의 complexity를 최소화하며 학습하는 ViT 구조를 제안
- 본 논문에서 제안하는 알고리즘인 X-ViT는 기존의 SA에 대해 nonlinearity를 제거한 모델 
- 기존의 ViT 코드에서 몇 줄만 변경했음에도 ImageNet Top-1 accuracy에 대해 Swin 모델, DeiT 모델에 비해 향상된 결과를 보임 

![image](https://user-images.githubusercontent.com/49440852/177773579-c09a2c10-0246-4141-8279-67ce72f62fcb.png)
